src/Microsoft.OData.Edm/ExtensionMethods/ExtensionMethods.cs

```diff
@@ -2027,6 +2027,18 @@ public static void AddAlternateKeyAnnotation(this EdmModel model, IEdmEntityType
             model.SetVocabularyAnnotation(annotation);
         }
 
+        /// <summary>
+        /// Adds a vocabulary annotation to this model.
+        /// </summary>
+        /// <param name="model">The model the annotation is added to.</param>
+        /// <param name="annotation">The annotation to be added.</param>
+        /// <param name="location">The location to add the allocation</param>
+        public static void AddVocabularyAnnotation(this EdmModel model, IEdmVocabularyAnnotation annotation, Csdl.EdmVocabularyAnnotationSerializationLocation location)

```

Comment:
>IEdmVocabularyAnnotation annotation, Csdl.EdmVocabularyAnnotationSerializationLocation location [](start = 72, length = 95)

It's not a deal for me to add such extensions. Because, there's only two line codes .

but, i have a question here:   How can handle this scenario: 

annotation itself has the Location setting, and that setting is different with the "location" given?

Reply:
I wasn't aware the IEdmVocabularyAnnotation  has a location. Will look into it.